addons/website/views/snippets/s_picture.xml

```diff
@@ -10,7 +10,7 @@
             <div class="row s_nb_column_fixed">
                 <div class="col-lg-10 offset-lg-1 pb24" style="text-align: center;">
                     <figure class="figure">
-                        <img src="/web/image/website.s_picture_default_image" class="figure-img img-thumbnail padding-large" alt=""/>
+                        <img src="/web/image/website.s_picture_default_image" class="figure-img img-fluid img-thumbnail padding-large" alt=""/>
```

Comment:
@can-odoo Could you fix it somewhere other than the xml file instead as this is a fix in stable.

Reply:
**Adding** a class in stable is fine, so it should stay like that. But indeed you can make a JS patch alongside this if the fix seems important. Either by patching computeSnippetTemplate, onBuilt, or in this case maybe even patching the option so that when you remove img-thumbnail, img-fluid is added ?

Reply:
Hello @qsm-odoo,
By following the other snippets(s_text_image), we are not removing the `img-fluid` class when adding `img-thumbnil`, `img-fluid` is there in any case, so here I have following the same and patching the options so that the class is available for a stable version. And patching can be removed in the master.

Reply:
I said adding a class in stable is fine... but not adding a template, adding a file in an asset, etc etc :grimacing: So here, I don't think patching `onBuilt` is the way to go since there is no existing option. Try doing what I suggested instead:

> patching the option so that when you remove img-thumbnail, img-fluid is added

Thanks :+1:

Reply:
Hello @qsm-odoo, can you please check? This is what we are looking for❓
This seems too specific only for `thumbnail`,  but I can't think of any other ways.